docs/index.rst

```diff
@@ -0,0 +1,37 @@
+Scylla Monitor
+=================
+
+.. toctree::
+   :hidden:
+   :maxdepth: 2 
+   
+   Scylla Monitor Support Matrix <matrix>
+   Install Scylla Monitor Stack <monitoring_stack>
+   The start-all.sh script <start_all>
+   Scylla Monitor Interfaces <monitoring_apis>
+   Deploy Scylla Monitor Without Docker <monitor_without_docker>
+   CQL Optimization Dashboard <cql_optimization>
+   Adding and Modifying Dashboards <updating_dashboard>
+   Alerting <alerting>
+   Troubleshoot Monitor <monitor_troubleshoot>
+
+.. include:: /_common/monitor-description.rst
+
+The Scylla Monitor Stack consists of three components, wrapped in Docker containers:
+
+* `prometheus` - collects and stores metrics
+* `alertmanager` - handles alerts
+* `grafana` - dashboard server
+
```

Comment:
For older version of Scylla Monitoring see `here <https://docs.scylladb.com/operating-scylla/monitoring/>`_